apps/core/validators.py

```diff
@@ -16,3 +16,17 @@ def name_validator(lastname):
             "ФИО может содержать только буквы русского или английского алфавита, "
             "между ними может присутствовать тире и пробел"
         )
+
+
+def nickname_validator(lastname):
```

Comment:
Я бы оставил старую функцию и просто расширил её функционал до цифр. А что, если кто-то имеет имя `Вильгельм 2`, тут уже ничего не поделаешь, да и валидационные ошибки будут разные, хотя и там и там про ФИО говорится. Но так тоже сгодится.) если qa не начнут ругаться) 
```suggestion
def nickname_validator(name):
```